app/src/pages/admin/projectsPage/projectName/projectName.jsx

```diff
@@ -73,13 +106,13 @@ export class ProjectName extends Component {
   render() {
     const {
       project: { projectSlug, organizationSlug, projectName },
-      isAssigned,
+      isProjectAssigned,
     } = this.props;
 
     return (
       <Link
         className={cx('name', {
-          'mobile-disabled': !isAssigned,
+          'mobile-disabled': !isProjectAssigned,
```

Comment:
This can be removed as we now allow visit unassigned projects.

Reply:
Done.